h2o-r/h2o-package/R/gam.R

```diff
@@ -103,11 +102,12 @@
 #'        Defaults to 0.
 #' @param max_runtime_secs Maximum allowed runtime in seconds for model training. Use 0 to disable. Defaults to 0.
 #' @param custom_metric_func Reference to custom evaluation function, format: `language:keyName=funcName`
-#' @param k Number of knots for gam predictors
-#' @param knots_keys String arrays storing frame keys of knots.  One for each gam column specified in gam_x
+#' @param num_knots Number of knots for gam predictors
+#' @param knot_ids String arrays storing frame keys of knots.  One for each gam column specified in gam_x
```

Comment:
same here:
One for each GAM column specified in gam_columns

Reply:
Fixed that too!